modules/nf-core/nonpareil/tests/main.nf.test

```diff
@@ -0,0 +1,93 @@
+nextflow_process {
+
+    name "Test Process NONPAREIL"
+    script "../main.nf"
+    process "NONPAREIL"
+    config "./nextflow.config"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "nonpareil"
+
+    test("candidatus_portiera_aleyrodidarum - gzipped") {
+
+        when {
+            params {
+                // define parameters here. Example:
+            // outdir = "tests/results"
+            }
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    file(params.test_data['candidatus_portiera_aleyrodidarum']['illumina']['test_1_fastq_gz'], checkIfExists: true)
+                ]
+                input[1] = 'fastq'
+                input[2] = 'kmer'
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                        file(process.out.npa[0][1]).name,
+                        file(process.out.npc[0][1]).name,
+                        path(process.out.npo[0][1]).readLines()[0],
+                        path(process.out.npl[0][1])
```

Comment:
assert snapshot(process.out.npa, ...) is not stable enough?

Reply:
npa/npc files change a lot (slightly stochastic numbering), as is npo.

Unless Id on't understand what you mean by 'stable enough'?

Reply:
that's exactly what I meant with stable enough. so good to go!